### ISSUE TITLE
[Copyright statement] Revoke the PR #157 for the copyright statement correction

### ISSUE THREAD

**Issue by MoonkiHong** [lf-edge/edge-home-orchestration-go#175]
_Regarding PR: #157 suggested by @tdrozdovsky, accepted by me (@MoonkiHong) and @suresh-lc:

We have deeply validated the copyright statement policy from open source communities especially projects in **LF Edge**, and our final decision from the evaluation is to revoke the PR #157 to the original one to state the latest released year for the contributed source codes. So I would like to report as the required item to be fixed as soon as possible. I am going to present a regarding PR shortly.

**Comment by tdrozdovsky** [comment]
@MoonkiHong I am a little surprised with this issue. Please provide me with a document that describes the correct date (for my understanding).
I will provide several sources that describe my assumption below:

https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html
https://opensource.google/docs/copyright/

**Comment by MoonkiHong** [comment]
@tdrozdovsky Your references are just references for other cases, but the existing statement is quite different from other LF Edge projects. Please see this: https://github.com/edgexfoundry/app-functions-sdk-go/commit/1034e843966f0cfc729a3c3cb90a33a655031538#diff-4dc29fe4b21f98a3b1a4218ee6b0385ac736428b4c1b9a7d6c5d4b33521b7d33.

**Comment by tdrozdovsky** [comment]
@MoonkiHong This is possibly a human error. That's why I requested the document.

How about such a link:
https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/
quote from this document
"By using a common format, the project avoids having to maintain lists of names of the authors or copyright holders, years or ranges of years, and variations on the (c) symbol. "

Do not misunderstand me, I want to know how to indicate correctly, because today the world has turned upside down for me. )))

**Comment by tdrozdovsky** [comment]
@MoonkiHong As far as I understand, copyright includes the date of creation + 50 years; in cases of file changes, indicate additional years to fix these changes, but at the same time do not delete the time of copyright registration.

P.S. I will follow the rules that you specify as you represent Samsung and you specify the copyright. I only pay attention to it

**Comment by MoonkiHong** [comment]
> @MoonkiHong This is possibly a human error. That's why I requested the document.
> 
> How about such a link:
> https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/
> quote from this document
> "By using a common format, the project avoids having to maintain lists of names of the authors or copyright holders, years or ranges of years, and variations on the (c) symbol. "
> 
> Do not misunderstand me, I want to know how to indicate correctly, because today the world has turned upside down for me. )))

@tdrozdovsky Yes, better to publish that official guideline document, but I do think that your request should be handled by a separate issue report, and properly bring it to the regarding organization if necessary. This is firstly for the resolution in conjunction with PR #176 though.

**Comment by MoonkiHong** [comment]
> @MoonkiHong As far as I understand, copyright includes the date of creation + 50 years; in cases of file changes, indicate additional years to fix these changes, but at the same time do not delete the time of copyright registration.
> 
> P.S. I will follow the rules that you specify as you represent Samsung and you specify the copyright. I only pay attention to it

@tdrozdovsky  Thank you for your attention. Yours is very reasonable and I am also very aware of it. Do not worry.

**Comment by tdrozdovsky** [comment]
@MoonkiHong Yes, I really care about that, since I have experience in copyright litigation.

Maybe you would ask this question to Samsung intellectual property lawyers.
